selfdrive/car/tesla/PCC_module.py

```diff
@@ -195,6 +197,13 @@ def max_v_by_speed_limit(self,pedal_set_speed_ms ,speed_limit_ms, CS):
       return pedal_set_speed_ms
 
   def reset(self, v_pid):
+    # Save the v_pid at every disengagement - to keep the nice smoothenss across drives.
+    if not RESET_PID_ON_DISENGAGE:
+      data = {}
+      data['v_pid'] = self.v_pid
+      with open(V_PID_FILE , 'w') as outfile :
+        json.dump(data, outfile)

```

Comment:
Is there a reason this needs to be persisted to disk? why not just store in a class variable?

Reply:
Yes, the idea is to re-use the last v_pid when engaging the first time - also across reboots/removal of eon in the car

Reply:
Ah, is this because you expect this process to restart while the car is in motion?
In any case, consider encapsulating the save/load into their own methods, this way it would be a bit more clear, and easier to move where you load / save, add new save points, etc.